### ISSUE TITLE
AWS China region

### ISSUE THREAD

**Issue by plribeiro3000** [fog/fog-aws#25]
Original issue at: fog/fog#3214

I don't think I see `cn-north-1` references in fog yet.

It was announced Dec 2013 http://aws.amazon.com/blogs/aws/coming-soon-new-china-beijing-region/

Services:

```
Amazon Elastic Compute Cloud (Amazon EC2)
Amazon Elastic Block Store (Amazon EBS)
Amazon Simple Storage Service (Amazon S3)
Amazon Relational Database Service (Amazon RDS)
Amazon DynamoDB
Amazon Elastic MapReduce (Amazon EMR)
Amazon Virtual Private Cloud (Amazon VPC)
Amazon CloudWatch
AWS CloudFormation
Amazon Simple Queue Service (Amazon SQS)
Amazon Simple Notification Service (Amazon SNS)
Auto Scaling
Elastic Load Balancing
Amazon Glacier
Amazon Simple Workflow (SWF)
AWS Identity and Access Management (IAM)
Amazon ElastiCache
AWS Storage Gateway
AWS Management Console
AWS Premium Support
```

**Comment by plribeiro3000** [comment]
cc/ @drnic @davetchen

**Comment by 1ein** [comment]
Hello, any progress with China region ? Thanks.

**Comment by geemus** [comment]
None that I'm aware of, but we would certainly welcome a PR relating to it.

**Comment by ssurenr** [comment]
Endpoint property can be used for now.
```ruby
connection = Fog::Storage.new({
  :provider                 => 'AWS',
  :aws_access_key_id        => YOUR_AWS_ACCESS_KEY_ID,
  :aws_secret_access_key    => YOUR_AWS_SECRET_ACCESS_KEY
  :endpoint                 => https://s3.cn-north-1.amazonaws.com.cn
})
```

**Comment by ssurenr** [comment]
> This issue has been marked inactive and will be closed if no further activity occurs.

Don't

**Comment by ssurenr** [comment]
Fixed via #169 and available in the latest release. You can close this @plribeiro3000

**Comment by geemus** [comment]
@ssurenr thanks for confirming it's fixed, closing.
